{{ cookiecutter.repo_name }}/Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM prefecthq/prefect:{{ cookiecutter.prefect_docker_image_tag }}
+FROM prefecthq/prefect:2.5.0-python3.9
 COPY requirements.txt .
+COPY add_nexus.sh .
+RUN chmod +x ./add_nexus.sh
+RUN ./add_nexus.sh
 RUN pip install -r requirements.txt
-ADD flows /opt/prefect/flows
\ No newline at end of file
+ADD flows /opt/prefect/flows
+ADD data /opt/prefect/data
```

Comment:
Die data folder is niet altijd nodig? Allez, de template maakt die niet aan.

Reply:
Idem hier.